pkgs/development/tools/misc/sipp/default.nix

```diff
@@ -1,29 +1,30 @@
-{lib, stdenv, fetchurl, ncurses, libpcap }:
+{ lib, stdenv, fetchurl, ncurses, libpcap, cmake, openssl, git }:
 
 stdenv.mkDerivation rec {
-  version = "3.6.0";
+  version = "3.6.1";
 
   pname = "sipp";
 
   src = fetchurl {
-    url = "https://github.com/SIPp/${pname}/releases/download/v${version}/${pname}-${version}.tar.gz";
-    sha256 = "1fx1iy2n0m2kr91n1ii30frbscq375k3lqihdgvrqxn0zq8pnzp4";
+    url =
+      "https://github.com/SIPp/${pname}/releases/download/v${version}/${pname}-${version}.tar.gz";
+    sha256 = "sha256-alYOg6/5gvMx3byt+zvVMMWJbNW3V91utoITPMhg7LE=";
   };
 
-  postPatch = ''
-    sed -i "s@pcap/\(.*\).pcap@$out/share/pcap/\1.pcap@g" src/scenario.cpp
+  configurePhase = ''
+    cmake . -DUSE_GSL=1 -DUSE_PCAP=1 -DUSE_SSL=1 -DUSE_SCTP=0
   '';
 
-  configureFlags = [
-    "--with-pcap"
-  ];
+  buildPhase = ''
+    make -j4
+  '';
 
-  postInstall = ''
-    mkdir -pv $out/share/pcap
-    cp pcap/* $out/share/pcap
+  installPhase = ''
+    mkdir -p $out/bin
+    mv sipp $out/bin
   '';
 
-  buildInputs = [ncurses libpcap];
+  buildInputs = [ ncurses libpcap cmake openssl git ];
```

Comment:
```suggestion
  buildInputs = [
    libpcap
    ncurses
    openssl
  ] ++ lib.optionals (stdenv.isLinux) 
    lksctp-tools
  ];
```

Reply:
Nice additions. Should I either incorporate those in my commit and add you as co-author or rather commit your suggestions from github? I'm new to this kind of workflow 😄🤷‍♂️

Reply:
Feel free to add me as co-author.

> Co-Authored-By: Martin Weinelt <hexa@darmstadt.ccc.de>